<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="GLWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWireframe"/>
   <addaction name="actionShader"/>
   <addaction name="actionNormals"/>
   <addaction name="actionAnimation"/>
   <addaction name="actionSkydome"/>
   <addaction name="actionTextures"/>
   <addaction name="actionAxis"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionShader">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shader</string>
   </property>
   <property name="toolTip">
    <string>Shader</string>
   </property>
  </action>
  <action name="actionNormals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normals</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
  <action name="actionSkydome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Skydome</string>
   </property>
   <property name="toolTip">
    <string>Skydome</string>
   </property>
  </action>
  <action name="actionTextures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Textures</string>
   </property>
   <property name="toolTip">
    <string>Textures</string>
   </property>
  </action>
  <action name="actionAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Axis</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
